### PR TITLE
Fix permissions for menu items

### PR DESCRIPTION
**Pull request by lassebm**
Tested with NetBox 3.6.8. Ref: https://github.com/netbox-community/netbox/blob/fedcbaf4c883aa7bc9ac51618ed55af30df14209/netbox/utilities/permissions.py#L27

### CODE REVIEW

**Comment by devon-mar** [comment]
Thanks for the PR! Can you add a test for this? (i.e. when the user does not have add permission they don't see the button)

You could base it off the following test:
https://github.com/devon-mar/netbox-kea/blob/d8edd01883a8d3e4452c258d73186df7a3b85513/tests/test_ui.py#L432

File: tests/test_ui.py [hunk 1]
```diff
@@ -321,10 +321,34 @@ def leases4_250(kea: KeaClient) -> None:
...
+
+
 @pytest.fixture(scope="function", autouse=True)
 def netbox_login(
-    page: Page, netbox_url: str, netbox_username: str, netbox_password: str
+    page: Page,
+    netbox_url: str,
+    netbox_username: str,
+    netbox_password: str,
+    netbox_user_permissions: list[dict],
```
Comment (devon-mar) [inline tests/test_ui.py:335]: nit: could you change all `list[dict]` to `list[dict[str, Any]]` or `list[dict[str, list[Any]]]`

**Review by devon-mar — CHANGES_REQUESTED** [review]
Looks good! Just a couple of nits.

File: tests/test_ui.py (hunk 1 above)
Comment (lassebm) [reply tests/test_ui.py]: Sure 👍

**Pull request merged**